.github/workflows/multichain-dev.yml

```diff
@@ -0,0 +1,59 @@
+name: Deploy Multichain Dev.
+on:
+  pull_request:
+    types: [closed]
```

Comment:
should we differentiate between closed and merged?

Reply:
That's what line 14 and 34 do 😄